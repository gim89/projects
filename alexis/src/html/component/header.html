<header class="head">
  //= nav.html
  <div class="container">
    <div class="head__promo section__text">
      <h1 class="head__title">We design things with love and passion.</h1>
      <a href="#" class="head__btn btn btn--hover btn--active">our works</a>
      <div class="head__icons">
          <figure class="head__figure">
            <img class="head__img" src="img/head-one.png" alt="">
            <figcaption class="head__figcaption color__green">fixed price projects</figcaption>
          </figure>
          <figure class="head__figure">
              <img class="head__img" src="img/head-two.png" alt="">
              <figcaption class="head__figcaption color__purple">receive on time</figcaption>
            </figure>
            <figure class="head__figure">
                <img class="head__img" src="img/head-three.png" alt="">
                <figcaption class="head__figcaption color__orange">satisfaction guranteed</figcaption>
              </figure>
        </div>
    </div>
  </div>
</header>