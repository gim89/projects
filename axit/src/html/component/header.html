<header class="head">
  //= nav.html
  <div class="head__container container">
    <section class="head__content">
      <h1 class="head__logo">
        <span class="head__span">ax</span>
        it
      </h1>
      <h2 class="head__title head__title--after">modern axure template for beautiful prototypes</h2>
      <p class="head__text">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
        Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus
        accumsan et viverra justo commodo.
      </p>
      <button class="head__btn btn btn--transparent btn--hover btn--active">Download</button>
    </section>
    //= signup.html
  </div>
</header>