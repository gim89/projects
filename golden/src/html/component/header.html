<header class="head">
  <div class="container">
    //= nav.html
    <div class="head__promo">
      <h2 class="head__title">Welcome To Our Studio!</h2>
      <h1 class="head__caption">It`s nice to meet you</h1>
      <a href="#" class="head__btn btn btn--hover btn--active">Tell me more</a>
    </div>
  </div>
</header>
